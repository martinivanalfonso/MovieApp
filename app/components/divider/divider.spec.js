import React from 'react'
import { shallow } from 'enzyme'
import Divider from './divider'

describe('Divider Component', () => {
  test('renders correctly', () => {
    const component = shallow(<Divider />)
    expect(component).toMatchSnapshot()
  })
})
