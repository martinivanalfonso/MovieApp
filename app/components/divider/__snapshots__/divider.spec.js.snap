// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Component renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#3A3A3A",
      "borderBottomWidth": 0.5,
      "margin": 14,
    }
  }
/>
`;
